test/fixtures/user/user.js

```diff
@@ -29,6 +29,10 @@ module.exports = () => {
       },
       status: "active",
       profileURL: "https://abcde.com",
+      picture: {
+        publicId: "profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar",
+        url: "https://res.cloudinary.com/realdevsquad/image/upload/v1667685133/profile/mtS4DhUvNYsKqI7oCWVB/aenklfhtjldc5ytei3ar.jpg",
```

Comment:
Added picture field in users fixture because we were getting it in the Prod but there was no test for it.